sleep/sleep_windows.go

```diff
@@ -0,0 +1,34 @@
+package sleep
+
+import "C"
+import (
+	"github.com/mysteriumnetwork/gowinlog"
+	"github.com/rs/zerolog/log"
+)
+
+func (n Notifier) Start() {
+	log.Debug().Msg("Register for sleep log events")
+
+	watcher, err := winlog.NewWinLogWatcher()
+	if err != nil {
+		log.Error().Msgf("couldn't create log watcher: %v\n", err)
+		return
+	}
+
+	watcher.SubscribeFromNow("System", "*[System[Provider[@Name='Microsoft-Windows-Power-Troubleshooter'] and EventID=1]]")
+	for {
+		select {
+		case <-watcher.Event():
+			n.eventbus.Publish(AppTopicSleepNotification, EventWakeup)
+		case err := <-watcher.Error():
+			log.Error().Msgf("Log watcher error: %v\n", err)
+		}
+	}
+	<-n.stop
```

Comment:
This `n.stop` should be in the loop and in the `select` block. Otherwise, this loop will never finish.